More Items

Macheya opened the pull request:
Made an item list for the silver ingot, not sure if this is the correct format, if it is let me know and I'll fix it.

Macheya committed:
More Items

Made an item list for the silver ingot, not sure if this is the correct format, if it is let me know and I'll fix it.

DarkMorford commented:
In general, your syntax is correct, but do note that modded items don't always follow the same pattern as vanilla ones. We'd need to check the language files for the mod silver comes from to make sure we get the right key. Also, you have some plus signs at the beginning of your lines that need to be removed.

I'm not sure where the "silver" to "silva" change comes from. Mispronouncing "gold" as "glod" is an Alex-ism, but I'm not familiar with a silver variant. I could just be mis-remembering, though. :P

DarkMorford commented:
Actually, looking at some other resource packs out there, modded stuff might need to go into its own set of folders (e.g. /assets/deepresonance instead of /assets/minecraft). Guess we'll just have to test it.

Macheya committed:
Just a thought.

You can get rid of this pull request if you want, I figured since "glod" was a funny mispronunciation we could do the same for the other ores, but nothing funny comes to mind for silver so I don't think it's necessary, especially since we're only replacing the things we want to change. I'll finish editing the lang file and copy all the stuff I decided to change in a later request.

DarkMorford commented:
Closing this PR per discussion.
